sdk/storage/storage-file-share/src/Clients.ts

```diff
@@ -5116,6 +5316,78 @@ export class ShareFileClient extends StorageClient {
 
     return appendToURLQuery(this.url, sas);
   }
+
+  /**
+   * Renames a file.
+   * This API does not support renaming a file from one share to another, or between storage accounts.
```

Comment:
>one share to another, or between storage accounts [](http://example.com/codeflow?start=52&length=49)

same here as above

Reply:
It only supports renaming in the same share.